src/Modules/CalcDefence.lua

```diff
@@ -1169,13 +1169,19 @@ function calcs.defence(env, actor)
 		end
 	end
 	
+	-- Life Recoverable
+	output.LifeRecoverable = output.LifeUnreserved
+	if env.configInput["conditionLowLife"] then
+		output.LifeRecoverable = m_min(output.Life * data.misc.LowPoolThreshold, output.LifeUnreserved)
+	end
+	
 	-- Prevented life loss (Petrified Blood)
 	do
 		output["preventedLifeLoss"] = modDB:Sum("BASE", nil, "LifeLossBelowHalfPrevented")
 		local portionLife = 1
-		if not modDB:Flag(nil, "Condition:LowLife") then
+		if not env.configInput["conditionLowLife"] then
 			--portion of life that is lowlife
-			portionLife = m_min(output.Life * data.misc.LowPoolThreshold / output.LifeUnreserved, 1)
+			portionLife = m_min(output.Life * data.misc.LowPoolThreshold / output.LifeRecoverable, 1)
```

Comment:
Isnt this incorrect now? As it ends up being `(output.Life * data.misc.LowPoolThreshold) / (output.Life * data.misc.LowPoolThreshold)`

Reply:
Life recoverable is only set to "output.Life * data.misc.LowPoolThreshold" when the lowlife condition is met, and this code only runs when its not met, I can change it back but it should always still be LifeUnreserved in this part of the code